ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -893,6 +893,7 @@ class EclCmdRun : public EclCmdWithEclTarget
             break;
         default:
             fprintf(stderr, "%s %s\n", respwuid.str(), resp->getState());
+            ret = 1;
```

Comment:
Right, I'll expand on the error codes.  I'll also probably expand on the text.  That was recently added because users who probably aren't checking return codes also get confused in some circumstances.